java/client/src/org/openqa/selenium/support/ui/Select.java

```diff
@@ -305,4 +306,21 @@ private void setSelected(WebElement option, boolean select) {
       option.click();
     }
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
```

Comment:
Nowadays, we tend to follow a pattern of:

```
if (!(o instanceof Select)) {
  return false;
}
Select that = (Select) o;
return Objects.equals(this.element, that.element);
```

Reply:
The performance is dramatically different, but we strip more boilerplate this way.

Reply:
@shs96c updated to follow project patterns